admin/views/pupil-pin/view-and-custom-print-pins.ejs

```diff
@@ -96,16 +96,18 @@
                                 <% } %>
                                 <input type="hidden" id="pupilUpn" name="pupilUpn" value="<%= pupil.upn %>"/>
                                 <div>
-                                    <div class="pin-content">
-                                        <span class="label">School Password: </span>
-                                        <span class="information"><%= (!!school ? school.pin : 'No active password') %></span>
-                                        <span class="separator"></span>
-                                        <span class="label">PIN: </span>
-                                        <span class="information">
-                                          <%= pupil.pin %>
+                                  <div class="pin-content">
+                                      <span class="label">School Password: </span>
+                                      <span class="information" id="schoolPin" name="schoolPin">
```

Comment:
The `name` attribute is not a valid attribute for `span`.  In any case, `id` should be enough?